compiler/ballerina-parser/src/test/resources/types/simple-types/simple_types_source_01.bal

```diff
@@ -12,4 +12,5 @@ public function foo() {
     service k;
     var l;
     never m;
+    () n;
```

Comment:
Shall we add some negative test as well? such as missing one of the parenthesis, additional tokens between the parenthesis, etc?

Reply:
sure

Reply:
@SupunS I have added more test cases.